assets/images/chevron-left.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 8 13" fill="none" xmlns="http://www.w3.org/2000/svg">
+	<path d="M7 1.00002L2 6.50002L7 12" stroke="#1E1E1E" stroke-width="1.5"/>
```

Comment:
```suggestion
	<path d="M7 1.00002L2 6.50002L7 12" stroke="currentColor" stroke-width="1.5"/>
```

Let's make sure these pick up the current text color, as the common practice we picked up from Gutenberg — makes it easier to color these icons in CSS.

Reply:
Wow, didn't know about this one. Thanks! Updated. 👍